tests/integration/conftest.py

```diff
@@ -56,10 +59,13 @@ def assert_cairo_test_cases(
     )
 
     if expected_broken_test_cases_names is None:
-        actual_broken = len(testing_summary.broken)
+        actual_broken = len(testing_summary.broken_suites) + len(testing_summary.broken)
```

Comment:
up

Reply:
Why is `actual_broken` a number here, but 2 lines below is a set?

Reply:
Refactored this function